autoscheduler/frontend/templates/index.html

```diff
@@ -18,10 +19,10 @@
 
         <!-- Dependencies -->
         <!-- I think we need these? -->
-        <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
-        <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+        <script crossorigin src="https://cdnjs.cloudflare.com/ajax/libs/react/16.12.0/umd/react.development.js"></script>
```

Comment:
I had to update the CDN because the old one wasn't working for some reason at one point during development. It seems to be working again now, so we could probably switch back if we wanted to.